src/pinning.js

```diff
@@ -201,9 +223,13 @@ class Pinning {
 
   _onMessage (topic, data) {
     console.log(topic, data)
-    if (data.type === 'PIN_DB' && OrbitDB.isValidAddress(data.odbAddress)) {
-      this.openDB(data.odbAddress, this._openSubStoresAndSendHasResponse.bind(this), this._openSubStores.bind(this))
-      this.analytics.trackPinDB(data.odbAddress)
+    if (OrbitDB.isValidAddress(data.odbAddress)) {
+      if (data.type === 'PIN_DB') {
+        this.openDB(data.odbAddress, this._openSubStoresAndSendHasResponse.bind(this), this._openSubStores.bind(this))
+        this.analytics.trackPinDB(data.odbAddress)
+      } else if (data.type === 'SYNC_DB' && data.thread) {
```

Comment:
in future does sync db have to check and handle threads differently?

Reply:
Yeah, I think soo. (see the updated pinning-room spec)